crates/pop-cli/src/commands/install/frontend.rs

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use crate::{
+	cli::{self, traits::Confirm},
+	commands::install::run_external_script,
+	common::binary::SemanticVersion,
+};
+use anyhow::{Result, anyhow};
+use duct::cmd;
+use std::path::PathBuf;
+
+const MIN_NODE_VERSION: u8 = 20;
+const NVM_INSTALL_SCRIPT: &str = "https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.3/install.sh";
```

Comment:
I think this should be more sophisticated by always fetching the latest release. For that we should do some digging as we currently do with polkadot-sdk releases. Anyway, not a stopper for this PR to be merged, but rather something for the backlog.